cmd/version/main.go

```diff
@@ -25,6 +35,26 @@ func main() {
 		log.Fatal("Could not load config options.", err)
 	}
 
+	if !(args.OUT == ""){
+		versionJSONPath := util.ShouldAbs(filepath.Join(util.ShouldGetThisDir(), "..", "..", args.OUT))
+
+		versionSplice := strings.Split(util.GetAutograderFullVersion(),"-")
+
+		version:= Version{
+			Short: util.GetAutograderVersion(),
+			Hash : versionSplice[1],
+			Status: versionSplice[2],
+		}
+
+		err := util.ToJSONFileIndentCustom(&version,versionJSONPath,""," ")
+		if err != nil {
+			log.Error("Failed to write to the JSON file", err, log.NewAttr("path", versionJSONPath))
+	
+		}
+
+		return 
+	}
+
 	fmt.Println("Autograder")
 	fmt.Printf("Short Version: %s\n", util.GetAutograderVersion())
 	fmt.Printf("Full  Version: %s\n", util.GetAutograderFullVersion())
```

Comment:
I don't like how the API version is handled here instead of in util/version.
It would be good if they could be handled in the same place
(maybe the API version can also live in VERSION.json).